src/macros.jl

```diff
@@ -650,6 +650,18 @@ adds groups of constraints at once, in the same fashion as @constraint. The mode
     end)
 """ :(@constraints)
 
+@doc """
+    @variables(m, args...)
+
+Adds multiple variables to model at once, in the same fashion as `@variable` macro. The model must be the first argument, and multiple variables can be added on multiple lines wrapped in a `begin ... end` block. For example:
+
+    @variables(m, begin
+  	x
+        y[1:2] >= 0
+        z, Bin, (start = 0)
```

Comment:
Can you add a sentence describing the need for the parentheses as well please? :)

Reply:
done